contracts/src/core/EigenDACertVerifier.sol

```diff
@@ -32,7 +31,6 @@ contract EigenDACertVerifier is EigenDACertVerifierV1, EigenDACertVerifierV2 {
         EigenDACertVerifierV2(
             _eigenDAThresholdRegistry,
             _eigenDASignatureVerifier,
-            _operatorStateRetriever,
             _registryCoordinator,
             _securityThresholdsV2,
             _quorumNumbersRequiredV2
```

Comment:
on second thought this changes the function signature, and hence the abi of the contract. So PR title should be `refactor!` right? What's your semantic for using `!` or not?
Wondering whether we shouldn't at least for now keep the function signature the same even if we don't use the arguments. Until we've settled on a way to version/upgrade this contract.

Reply:
Only the operator state retriever getter is gone, which I highly doubt was being used. Contract constructor arguments are also different, but that is all. The interface to verify remains the same.

Reply:
Ah right I'm dumb this is just a breaking change on constructor, which is not important (you've updated the deployer script to work with this new constructor. So LGTM, thanks for explanation